cmd/controller-manager/app/controllers.go

```diff
@@ -184,6 +225,10 @@ func AddControllers(
 		controllers["storagecapability-controller"] = storageCapabilityController
 	}
 
+	if multiClusterEnabled {
+		controllers["globalrole-controller"] = globalRoleController
```

Comment:
I think we don't need globalrolebinding controller either when multi cluster is not enabled.